### ISSUE TITLE
More than one exoplanet

### ISSUE THREAD

**Issue by kmshariat** [HassanH01/Ideal_LCs#7]
Till now we have defined functions that can generate light curves for a single exoplanet system. We should work on writing functions that can generate Light Curves for more than one exoplanet system.

**Comment by hex-0x0** [comment]
like instead of getting the metrics for a single exoplanet, we should modify the code so that we can get a lightcurve for a system with multiple exoplanets?

**Comment by kmshariat** [comment]
yeah, like that. but not rewrite the whole code. We should create another function that can take no. of exoplanets as a parameter and generate light curve
